pkgs/data/fonts/u001/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "U001";
+  pname = "u001";
```

Comment:
IMO this is way too confusing, coupled with the fact that the package is essentially will be called `u001-font` in `all-packages.nix`. Maybe having a unified name would be better?

Reply:
I'm not really sure what the best option here is. I disagree with not adding `-ttf` or `-font` as these will run into naming collisions fast and fonts are a different kind of software. I can ask around for suggestions as well on what to do here.

Reply:
Though now that I think about it, this seems more like a `nixpkgs` org issue--that fonts should probably be @ `font.XXX` just like `haskellPackages.XXX`.